_posts/2025-03-10-cing-the-web.md

```diff
@@ -0,0 +1,227 @@
+---
+layout: post
+title: C-ing the web, a front-end story
+author: tatut
+excerpt: >
+  C is the grand old man of systems programming, still widely used today, but
+  not often used for web development. Let's try to make a client side front-end
+  with it.
+tags:
+ - C
+ - Frontend
+---
+
+The C programming language is the grand old man of systems programming.
+It is undeniably that C makes the world go around, being the implementation
+language for such "minor" things as the [Linux kernel](https://git.kernel.org/), [sqlite](https://sqlite.org/index.html), and [many](https://github.com/python/cpython) [programming](https://github.com/ruby/ruby) [language](https://github.com/SWI-Prolog/swipl-devel) [runtimes](https://github.com/Perl/perl5).
+
+Although C was used in the early web through [Common Gateway Interface](https://en.wikipedia.org/wiki/Common_Gateway_Interface),
+it has never been the go-to language for web development, especially not on the front end!
+
+With WebAssembly we can run any native code in the browser, so it is time to try
```

Comment:
I'd rephrase this somehow since WebAssembly is still a VM. "With WebAssembly browsers have gained a runtime friendly to languages like C"?